packages/plans-grid-next/src/hooks/data-store/types.ts

```diff
@@ -0,0 +1,23 @@
+import { FeatureList, PlanSlug, TERMS_LIST } from '@automattic/calypso-products';
+import { AddOnMeta } from '@automattic/data-stores';
+import { UseCheckPlanAvailabilityForPurchase } from '@automattic/data-stores/src/plans';
+import { HiddenPlans, PlansIntent } from '../../types';
+import { UseFreeTrialPlanSlugs } from './use-grid-plans';
+
+export interface UseGridPlans {
```

Comment:
Should this be `UseGridPlansProps` instead?

Reply:
Good catch, thanks. I tend to use `...Params` to differentiate hook parameters from component props. So in this case, `UseGridPlansParams`. Wdyt?

Reply:
`UseGridPlansParams` sounds good to me

Reply:
Updated: [2c055ec](https://github.com/Automattic/wp-calypso/pull/87478/commits/2c055ecd6787f93ba6907b39767e91f082f649ba)